build/yamls/antrea-sriov.yml

```diff
@@ -0,0 +1,3268 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
we cannot just check-in a yaml manifest for Antrea which is not auto-generated, it will just become outdated quickly and unusable. All manifests should be auto-generated by https://github.com/antrea-io/antrea/blob/main/hack/generate-manifest.sh. The script will start from some common base yaml files and merge them using kustomize, applying different patches for different use cases based on the command-line flags provided to the script. You can put the patches you require for SRIOV there: https://github.com/antrea-io/antrea/tree/main/build/yamls/patches. The `manifest` Makefile target takes care of regenerating all checked-in manifests by invoking `generate-manifest.sh` as many times as needed, with the correct flags.

Reply:
Thanks! Posted update to build/yamls/patches and generate-manifest.sh.